src/bin/vip-export-sql.js

```diff
@@ -6,12 +6,19 @@ import { makeCommandTracker } from '../lib/tracker';
 
 const examples = [
 	{
-		usage: 'vip export sql @mysite.develop',
-		description: 'Export SQL file from your site and save it to the current directory',
+		usage: 'vip @example-app.develop export sql',
+		description:
+			'Download a copy of the most recent database backup for an environment as an archived SQL file to the current local directory.',
```

Comment:
> Download a copy of the most recent database backup for an environment as an archived SQL file to the current local directory.

What do you think about this phrasing:

> Download an archived copy of the most recent database backup for an environment to the current local directory.

More concise and it doesn't duplicate SQL which is strongly implied by the command being `vip export sql`